pkg/datastore/datastore.go

```diff
@@ -14,22 +14,33 @@ import (
 type Datastore interface {
 	// This gets all clusters that match the color code
 	GetClusters(colorCodes []string) ([]types.SubmarinerCluster, error)
+
 	// This gets a single cluster based on cluster ID and returns a v1.Cluster or error based on what it retrieves
-	GetCluster(clusterID string) (types.SubmarinerCluster, error)
```

Comment:
Likewise, this is from another patch.

Reply:
Sorry for the noise, this should not be here, bad rebase.